### PR TITLE
Small support UI tweaks

### PR DESCRIPTION
**Pull request by fofr**
- Put email back into UI title, but not browser title following #661 
- Use a clearer reference status

### CODE REVIEW

File: app/views/support_interface/application_forms/show.html.erb [hunk 1]
```diff
@@ -14,6 +13,11 @@
   </div>
 <% end %>
 
+<h1 class="govuk-heading-xl">
```
Comment (tijmenb) [inline app/views/support_interface/application_forms/show.html.erb]: Can you add this to `content_for :title` for consistency?

Comment (fofr) [reply app/views/support_interface/application_forms/show.html.erb]: You can't set `:title` and add a caption AFAIK?

Comment (tijmenb) [reply app/views/support_interface/application_forms/show.html.erb]: This should work:

```rb
<% content_for :title do %>
  <span class="govuk-caption-xl">Application #<%= @application_form.id %></span>
  <%= @application_form.candidate.email_address %> 
<% end %>
```

Comment (fofr) [reply app/views/support_interface/application_forms/show.html.erb]: Fixed in 39f29e7

**Pull request merged**